### ISSUE TITLE
Exploring scroll-to-play functionality

### ISSUE THREAD

**Issue by danielpetty** [denverpost/dptv#183]
Is it possible to set up the videos so that it's scroll-to-play: If the user's screen shows the video (i.e., it's in the middle of the article) then the video autoplays once the user scrolls to that point.

**Comment by freejoe76** [comment]
You're talking about in-article iframed videos, right?

**Comment by danielpetty** [comment]
Correct, yes. I heard about this through Perform Media's sports highlights functionality. It only autoplays once the video appears on your screen, so it doesn't start autoplaying on article load. This would save us from having to create multiple freeforms for autoplay/non-autoplay.

**Comment by freejoe76** [comment]
Yea, gotcha. How's the autoplay experiement going? I thought we were only doing it for a week or two.

**Comment by danielpetty** [comment]
No complaints so far, I'm told. And nothing has been forwarded, so far so good? We'd need to look at bounce rate more closely, as you mentioned.

**Comment by freejoe76** [comment]
Finished a working draft of this functionality.

**Comment by freejoe76** [comment]
Loading the iframe and then playing it when it comes into view requires much more work than loading & playing the iframe when it comes into view.

**Comment by freejoe76** [comment]
Dan, if you can provide a good reason to load, and then play, I'll continue work on this.

**Comment by danielpetty** [comment]
My concern with the second option is that loading it all at once we take so long that people wouldn't see it at all.
